### PR TITLE
chore: add PR naming convention

### PR DESCRIPTION
**Pull request by KATT**
OK naming:

- chore: x
- fix(`x`)
- chore(`x`)
- feat(`x`)

### CODE REVIEW

**Review by bautistaaa — APPROVED** [review]
oh hell yea

File: .github/workflows/pr-naming.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+name: Pull Request naming convention
+# This workflow is triggered on pushes to the repository.
+on:
+  pull_request:
```
Comment (juliusmarminge) [inline .github/workflows/pr-naming.yml:4]: I think this must be a pull_request_rather if the workflow uses a secret and should be exposed to forks

Comment (juliusmarminge) [reply .github/workflows/pr-naming.yml:4]: We have something dimilar in t3: https://github.com/t3-oss/create-t3-app/blob/next/.github/workflows/pr-labeler.yml

Comment (KATT) [reply .github/workflows/pr-naming.yml:4]: it seems to work 🤷 

try renaming the PR

Comment (juliusmarminge) [reply .github/workflows/pr-naming.yml:4]: Im not on a form 😎

Comment (KATT) [reply .github/workflows/pr-naming.yml:4]: ohhhhh,

Comment (bautistaaa) [reply .github/workflows/pr-naming.yml:4]: the docs seem diff for both but i almost wonder should we use the one you're using on T3. seems more popular 🤣

Comment (KATT) [reply .github/workflows/pr-naming.yml:4]: https://github.com/trpc/trpc/pull/4380

**Pull request merged**